microcloud/cmd/microcloud/ask.go

```diff
@@ -520,6 +523,17 @@ func askNetwork(sh *service.ServiceHandler, peers map[string]mdns.ServerInfo, lx
 		return nil, nil, nil
 	}
 
+	if len(peers) != len(networks) {
+		wantsSkip, err := cli.AskBool("Some systems are ineligible for distributed networking. Continue anyway? (yes/no) [default=yes]: ", "yes")
```

Comment:
I now get this message when I have 4 VMs selected for MicroCloud, and when I confirm all 4 of them are displayed, so apparently they are all eligible?